recipes/django-bootstrap5/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "django-bootstrap5" %}
+{% set version = "21.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb7cbc1582e90718be7ba8e19cdafa47a5bbb102de9c7855b1a1299bae162c1d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.8
+    - django >=2.2
+    - beautifulsoup4 >=4.8.0
+
+test:
+  imports:
+    - django_bootstrap5
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/zostera/django-bootstrap5
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

The file part of the pypi package is `LICENSE`:
```suggestion
  license_file: LICENSE
```